libraries/AP_HAL_ChibiOS/Scheduler.cpp

```diff
@@ -623,6 +623,19 @@ void Scheduler::check_low_memory_is_zero()
         }
 #pragma GCC diagnostic pop
     }
+
+#if 0
+    /*
+      enable this on H7 to make writes to the first 1k of RAM on H7
+      produce a hard fault and crash dump
```

Comment:
```suggestion
      enable this on H7 to make writes to the first 1k of RAM on H7
      produce a hard fault and crash dump instead of an internal error.
      do not enable on production aircraft!
```